sources/packages/web/src/views/institution/InstitutionUserProfile.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <Message severity="info">
+    Please notice that the read-only information below is retrieved from your
+    BCeID account and it is not possible to change it here. If any read-only
+    information needs to be changed please visit
+    <a rel="noopener" href="https://www.bceid.ca/" target="_blank">bceid.ca</a>.
+  </Message>
+  <Card class="p-m-4">
+    <template #content>
+      <formio
+        formName="institutionUserProfile"
+        :data="initialData"
+        @submitted="submitted"
+      ></formio>
+    </template>
+  </Card>
+</template>
+
+<script lang="ts">
+import { ref, onMounted } from "vue";
+import { useRouter } from "vue-router";
+import { useToast } from "primevue/usetoast";
+import formio from "../../components/generic/formio.vue";
+import { UserService } from "../../services/UserService";
+import { InstitutionDto } from "../../types";
+import { InstitutionService } from "../../services/InstitutionService";
+import { InstitutionRoutesConst } from "../../constants/routes/RouteConstants";
+
+export default {
+  components: { formio },
+  setup() {
+    // Hooks
+    const toast = useToast();
+    const router = useRouter();
+    // Data-bind
+    const initialData = ref({});
+
+    const submitted = async (data: InstitutionDto) => {
+      let redirectHome = true;
+      try {
+        await InstitutionService.shared.updateInstitute(data);
```

Comment:
My understanding is that this screen is only updating the user email. This method is updating much other information for the institution and I believe that it will erase a lot of data on the institution side.
Can you please check if makes sense to use this method? It seems to me that the information should be saved using a specific method.

Reply:
IMO, this is user-specific info and it should be in the user service.

Reply:
Yes. Now user service is being used here.